Marlin/planner.cpp

```diff
@@ -1196,12 +1207,8 @@ void Planner::buffer_line(ARG_X, ARG_Y, ARG_Z, const float &e, float fr_mm_s, co
  *
  * On CORE machines stepper ABC will be translated from the given XYZ.
  */
-void Planner::set_position_mm(ARG_X, ARG_Y, ARG_Z, const float &e) {
-
-  #if PLANNER_LEVELING
-    apply_leveling(lx, ly, lz);
-  #endif
 
+void Planner::_set_position_mm(const float &lx, const float& ly, const float& lz, const float& e) {
```

Comment:
Minor point of style. Note that `&` and `*` should be tied to the variable name and not the storage type. So, `float &ly` and not `float& ly`.